src/html/services/index.html

```diff
@@ -0,0 +1,60 @@
+<div class="container" infinite-scroll="loadMore()" infinite-scroll-distance="5" infinite-scroll-use-document-bottom="true">
+    <a href="#!/services/add" id="newService" class="btn-floating btn-large waves-effect waves-light right">
+        <i class="material-icons">add</i>
+    </a>
+
+    <h3 class="header">Services</h3>
+
+    <p class="flow-text center" ng-show="total === null" style="margin-top:50px;">
+        <app-loader></app-loader>
+    </p>
+
+    <div class ng-show="total === 0">
+        <p class="flow-text center">
+            You haven't created any Services in Kong yet.
+        </p>
+        <p class="center">
+            <a href="#!/services/add" class="waves-effect waves-light btn">
+                <i class="material-icons left">add_box</i>
+                Add Service
+            </a>
+        </p>
+    </div>
+
+    <table class="bordered" ng-show="total > 0">
+        <thead>
+        <tr>
+            <th ng-repeat="prop in env.schemas.service.properties">{{prop.display_name}}</th>
```

Comment:
I wouldn't display all the information of a Service on this page. This leads to many columns and isn't very readable.

This is what it looks like in my browser:
<img width="900" alt="capture d ecran 2018-05-26 a 12 25 52" src="https://user-images.githubusercontent.com/6191583/40579592-0030a946-60e0-11e8-884a-696b63ea1ee5.png">

I would display collapse "protocol", "host", "port" and "path" into a single column named "url", and not display "updated at", "created at", and the various options (retry, timeouts...). The user can still click edit to see the details of a given Service.

Reply:
Removed the various options as suggested.

However, I did not collapse protocol, host, port, and path into URL since the user experience is better now after removing options. Please let me know if you think otherwise.

<img width="1099" alt="screen shot 2018-05-28 at 8 50 23 am" src="https://user-images.githubusercontent.com/3665346/40621999-3fa77346-6254-11e8-9ba0-99a6f344b4c2.png">

Reply:
Looks good to me